### DISCUSSION TITLE
CUSA24159 - Ghosts 'n Goblins Resurrection

### DISCUSSION THREAD

**Discussion by Zehkarluz** [GoldHEN/GoldHEN_Cheat_Repository#13] [Cheat requests]
Inf health, please. Thanks

**Comment by kmeps4** [comment]
Will check if there's a cheat available to add

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
